### PR TITLE
add models memory footprint to catalog

### PR DESCRIPTION
**Pull request by feloy**
### What does this PR do?

Adds memory footprint of models, as computed by the worflow `.github/workflows/compute-model-sizes.yaml`

### What issues does this PR fix or reference?

Part of #479

### CODE REVIEW

**Comment by feloy** [comment]
I forgot a factor of 1024 for values, I'll change ...

**Review by axel7083 — APPROVED** [review]
LGTM

File: packages/backend/src/assets/ai.json [hunk 1]
```diff
@@ -83,7 +83,8 @@
       "hw": "CPU",
       "registry": "Hugging Face",
       "license": "Apache-2.0",
-      "url": "https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF/resolve/main/mistral-7b-instruct-v0.1.Q4_K_M.gguf"
+      "url": "https://huggingface.co/TheBloke/Mistral-7B-Instruct-v0.1-GGUF/resolve/main/mistral-7b-instruct-v0.1.Q4_K_M.gguf",
+      "memory": 4370464768
```
Comment (jeffmaury) [inline packages/backend/src/assets/ai.json:87]: 4,07 * 1024 * 1024 * 1024 = 4 370 129 224 after rounding on my side

Comment (feloy) [reply packages/backend/src/assets/ai.json]: oh right, I've first rounded at the MiB, then converted to bytes. That should be ok now. Thanks

**Review by jeffmaury — APPROVED** [review]
LGTM

**Pull request merged**